BCStudentSoftwareDevTeam/celts
# Refactor volunteer view of event form

Issue #100 : The UI view for users who are volunteers still has input boxes even though they can not edit the contents. This is an enhancement of the view.

The following is how it looks like before. 

<img width="1134" alt="Screen Shot 2021-09-03 at 9 23 24 AM" src="https://user-images.githubusercontent.com/46546491/132012243-dc64c5fa-87cb-4157-a2f9-28b36b6cb9ac.png">

The following show the view looks like after. 
<img width="1072" alt="Screen Shot 2021-09-03 at 9 24 56 AM" src="https://user-images.githubusercontent.com/46546491/132012442-f85e5d77-4c4a-4b39-b7ae-b0b83036675f.png">

Test:

1. Login in as a student and navigate to /1/1/edit_event
2. The view should be the same as the second uploaded picture.
3. Login as a Celts students staff or Celts admin and navigate to /1/1/edit_event
4. The following view should be displayed.
<img width="1118" alt="Screen Shot 2021-09-03 at 9 28 27 AM" src="https://user-images.githubusercontent.com/46546491/132012973-cf0b801b-5d91-40f2-a2ee-dd247019693a.png">
